### PR TITLE
Fixed bug in image saver, modified gitignore, added support for KDE and added 

### PR DESCRIPTION
**Pull request by classPythonAddike**
Added the requested changes :eyes: Now tappetti creates `./data/images/` if it doesn't exist, before downloading the files.

### CODE REVIEW

**Review by Muhimen123 — CHANGES_REQUESTED** [review]
More fixes.

**Comment by classPythonAddike** [comment]
:ohno:

File: utils/image_saver.py [hunk 1]
```diff
@@ -38,8 +39,13 @@ def save_image() -> None:
         ) if answer == 'TID Repo' else input("Please enter download link: ")
 
         path: str = os.path.join(os.getcwd(), "data", "images")
+    
+        if not os.path.exists(path):
```
Comment (Muhimen123) [inline utils/image_saver.py]: This partially solve the problem for `save` command but the bug is still there for the `download` command. You can put this if statement in `downloader.py`. Should solve the problem. 

Put it somewhere around *here*.

image

**Comment by Muhimen123** [comment]
Also, can you remove the following line from the xfce part? 

```python
print(result)
```

I forgot to remove it earlier 😝

**Comment by classPythonAddike** [comment]
> Also, can you remove the following line from the xfce part?
> 
> ```python
> print(result)
> ```
> 
> I forgot to remove it earlier

Sure :eyes:

**Comment by classPythonAddike** [comment]
@Muhimen123 finished the changes :smilecat:

**Comment by Muhimen123** [comment]
> @Muhimen123 finished the changes :smilecat:

Thanks, let me take a quick look.

**Review by Muhimen123 — APPROVED** [review]
Works like a charm!

**Pull request merged**